src/index.js

```diff
@@ -20,4 +24,48 @@ const closebutton = document.getElementById('closebutton');
 closebutton.onclick = () => {
   modal.classList.add('scale-0');
   modal.classList.remove('scale-100');
-};
\ No newline at end of file
+};
+
+// Nav Action
+const tabLinks = document.querySelectorAll('.nav-link');
+const content = document.getElementById('content');
+
+const renderPage = (location, page) => {
+  location.innerHTML = '';
+  page();
+};
```

Comment:
### Elegant Implementation of a callback

I'd say it is a very elegant way of writing a single-page render. This is completely lightweight and scalable.